### PR TITLE
[Minor] Add more data and fix up some data issues

### PR DESCRIPTION
**Pull request by justinmclean**
Fix a few data issues and add some more data to be able to test things. Rename catalogs to have more consistent descriptive names.

### CODE REVIEW

File: init/postgres/init.sql [hunk 1]
```diff
@@ -93,19 +93,1167 @@ insert into hr.departments (department_id, department_name) values (1,'Sales');
...
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (5,3,'Training','2023-08-26','2023-08-28');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (6,3,'Training','2017-03-31','2017-04-03');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (7,3,'Training','2021-10-22','2021-10-25');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (8,3,'Training','2020-01-16','2020-01-16');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (9,3,'Training','2022-07-18','2022-07-18');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (10,3,'Training','2017-10-09','2017-10-09');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (11,3,'Training','2018-05-20','2018-05-23');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (12,3,'Training','2017-03-31','2017-04-03');
```
Comment (jerryshao) [inline init/postgres/init.sql:123]: I think you can use one insert with multiple values, no need to write many SQL queries here and below.

File: init/trino/init.sh [hunk 2]
```diff
@@ -23,21 +23,21 @@ else
 fi
 
 # Create Hive catalog for experience Gravitino service
-curl -X POST -H "Content-Type: application/json" -d '{"name":"catalog_demo","type":"RELATIONAL", "provider":"hive", "comment":"comment","properties":{"metastore.uris":"thrift://hive:9083"}}' http://gravitino:8090/api/metalakes/metalake_demo/catalogs
+curl -X POST -H "Content-Type: application/json" -d '{"name":"catalog_hive","type":"RELATIONAL", "provider":"hive", "comment":"comment","properties":{"metastore.uris":"thrift://hive:9083"}}' http://gravitino:8090/api/metalakes/metalake_demo/catalogs
```
Comment (jerryshao) [inline init/trino/init.sh:26]: Shall we also update the README, since the catalog name is changed?

**Comment by qqqttt123** [comment]
@justinmclean Do you test this shell in your local host?

**Comment by justinmclean** [comment]
Yes I tested this locally.

**Comment by qqqttt123** [comment]
We should also modify the documents of other places like https://github.com/datastrato/gravitino/blob/main/docs/how-to-use-the-playground.md and the website.

**Comment by qqqttt123** [comment]
I have a little concern about the initialization time of SQL if we add too many data.

**Comment by justinmclean** [comment]
There is no issue with  initialization time

**Comment by qqqttt123** [comment]
I verified the new patch in my local host. It's ok, too.

**Review by qqqttt123 — APPROVED** [review]
LGTM.

**Comment by jerryshao** [comment]
@qqqttt123 if there's no further comment, please merge it.

**Pull request merged**